camel-kafka-connector-catalog/src/main/java/org/apache/camel/kafkaconnector/catalog/CamelKafkaConnectorCatalog.java

```diff
@@ -83,7 +83,7 @@ private void initCatalog() {
         }
     }
 
-    public String getConnectorAsJson(String connectorName) {
+    private String loadConnectorAsJsonFromEmbeddedCatalog(String connectorName) {
```

Comment:
I wouldn't change this name. It doesn't add so much.

Reply:
The added difference is that the method is loading from embedded catalog, it is not able to provide the json from the cached connector that are dynamically added. With the name "getConnectorAsJson", I'm expecting to have also the dynamically added connector to be provided.

Reply:
Sounds good them